src/snippet.js

```diff
@@ -13,8 +13,7 @@ const snippet = (
     throw new Error('FullStory orgId is a required parameter');
   }
   /* begin FullStory snippet */
-  window['_fs_debug'] = debug;
```

Comment:
This is part of [the changes to use a separate `fs-debug.js` script](https://help.fullstory.com/hc/en-us/articles/360020829233-What-is-fs-debug-). We'll want to make a corresponding change in this SDK. We can merge this, I'll open an issue to track that change.